kd_sweep.sh

```
@@ -260,47 +260,49 @@ elif [[ "$MODE" == "run_all_4m" ]]; then
   # 9) SampledKD k=75
-  sbatch train.slurm top-k-tok 75 light "$KD_SWEEP_TAG"
+  sbatch --wait train.slurm top-k-tok 75 light "$KD_SWEEP_TAG"
 
   # 11) SampledKD Pos RS-KD k=25
   sbatch train.slurm pos-rs-kd 25 light "$KD_SWEEP_TAG"
 
   # # 12) SampledKD Trained also on GSM8K k=25 (GSM8K-only run)
   # # Note: train.slurm supports DATASETS / DATASET_CONFIG / PROMPT_COL / ANSWER_COL overrides
-  # sbatch --export=ALL,DATASETS="gsm8k",DATASET_CONFIG="main",PROMPT_COL="question",ANSWER_COL="answer" \
-  #   train.slurm top-k-tok 25 light "$KD_SWEEP_TAG"
+  sbatch --export=ALL,DATASETS="gsm8k",DATASET_CONFIG="main",PROMPT_COL="question",ANSWER_COL="answer" \
```

Shellcheck (0.11.0)

[warning] Word is of the form "A"B"C" (B indicated). Did you mean "ABC" or "A\"B\"C"?

(SC2140)